openmodelica/cppruntime/fmu/modelExchange/2.0/Crane_FMU2_CPP.mos

```diff
@@ -6,6 +6,11 @@
 
 loadModel(Modelica); getErrorString();
 setCommandLineOptions("+simCodeTarget=Cpp"); getErrorString();
+
+// disable directional derivatives as long as it fails
+// for this model
+setCommandLineOptions("+d=disableDirectionalDerivatives"); getErrorString();
```

Comment:
@wibraun Why do the directional derivatives fail?

Reply:
Because Cpp code generation needs some fixes, in this case it's about support of inlined jacobian in the jacobian system. In the other case I don't know since the code generation just fails without an error message.

Reply:
I see. Will you create a ticket to keep track of this issue?

Reply:
Yes, of course as far as it is pushed to the master.